WebKitBrowser/WebKitBrowser.cpp

```diff
@@ -350,6 +354,8 @@ namespace Plugin {
     {
         if (connection->Id() == _connectionId) {
 
+            SYSLOG(Logging::Notification, (_T("WebKitBrowser::Deactivated: { \"URL\": %.*s }"), 80, _lastURL.c_str()));
```

Comment:
Is there really an need to have this syslog for every webkit deactivation? Why not have it as a trace? 
If it needed can I propose to move it the PerformanceMetrics plugin?

Reply:
PerformanceMetrics is not available for R2. I can make it a TRACE if that is a problem.